.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
Noticed that not all the files are consistently using indent_size 2 some are using 4. Should it be 2 or 4.